pkgs/cli/src/main.zig

```diff
@@ -87,12 +92,37 @@ const ZeamArgs = struct {
                 };
             },
         },
+        lean_node: struct {
+            help: bool = false,
+            config_filepath: []const u8 = "./config.yaml",
```

Comment:
they can't be a default path, the files need to be dynamically generated for this command and the lean_node needs to be started with `--customGenesis <genesis dir path>`